drivers/peci/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
```

Comment:
use  zephyr_library_sources_ifdef

Reply:
That is done when XEC driver is introduced in a later commit. 
This is just generic subsystem folder